ax/runners/submitit.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python3
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Any, Dict, Iterable, Optional, Set
+
+import pandas as pd
+
+from ax.core.base_trial import BaseTrial, TrialStatus
+from ax.core.data import Data
+
+from ax.core.metric import Metric, MetricFetchE, MetricFetchResult
+from ax.core.runner import Runner
+from ax.core.trial import Trial
+from ax.utils.common.result import Err, Ok
+
+from submitit import (
+    AutoExecutor,
+    DebugExecutor,
+    DebugJob,
+    LocalExecutor,
+    LocalJob,
+    SlurmJob,
+)
+from submitit.core.core import Executor
+
+
+class SubmitItRunner(Runner):
```

Comment:
Are `poll_exception` and `poll_available_capacity` important to define for a Runner?